webrtc.html

```diff
@@ -3849,29 +3888,21 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       interface as described below.</p>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
-        <dt>void getStats(MediaStreamTrack? selector, RTCStatsCallback
-        successCallback, RTCPeerConnectionErrorCallback failureCallback)</dt>
+        <dt>Promise&lt;RTCStatsReport&gt; getStats(
+          optional MediaStreamTrack? selector)</dt>
 
         <dd>
           <p>Gathers stats for the given <a href="#stats-selector">selector</a>
           and reports the result asynchronously.</p>
 
           <p>When the <dfn id=
-          "dom-peerconnection-getstats"><code>getStats()</code></dfn> method is
-          invoked, the user agent MUST queue a task to run the following
-          steps:</p>
+          "#widl-RTCPeerConnection-getStats-Promise-RTCStatsReport--MediaStreamTrack-selector">
+            <code>getStats()</code></dfn> method is
+          invoked, the user agent MUST run the following steps:</p>
 
           <ol>
             <li>
-              <p>If the <code><a>RTCPeerConnection</a></code> object's <a href=
-              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
-              signalingState</a> is <code>closed</code>, throw an
-              <code>InvalidStateError</code> exception.</p>
-            </li>
```

Comment:
Or did you just drop it because we plan to keep some form of stats info alive even for closed channels?